### PR TITLE
add CopyGetter for some attributes which are copyable

### PR DESCRIPTION
**Pull request by bachue**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -43,14 +40,14 @@ fn main() {
 
 The above structure definition generates the following output with `cargo expand`.
 
-```rust,ignore
+```rust
```
Comment (Hoverbear) [inline src/lib.rs]: This needs to stay `ignore` for the CI to pass.
```suggestion
```rust,ignore
```

**Comment by Hoverbear** [comment]
@djc mentioned in a Twitter conversation that if specialization stabilizes we could use that. @djc Do you think you could be a second reviewer on this? Do you think it's worthwhile to consider a deprecation path for this?

File: src/generate.rs [hunk 2]
```diff
@@ -12,6 +12,7 @@ pub struct GenParams {
 #[derive(PartialEq, Eq)]
 pub enum GenMode {
     Get,
+    GetByCopy,
```
Comment (Hoverbear) [inline src/generate.rs]: In fitting with `GetMut` could we use `GetCopy`?

Comment (bachue) [reply src/generate.rs]: It's OK

File: src/generate.rs [hunk 3]
```diff
@@ -74,7 +78,9 @@ pub fn implement(field: &Field, mode: &GenMode, params: &GenParams) -> TokenStre
     let fn_name = Ident::new(
         &format!(
             "{}{}{}{}",
-            if has_prefix_attr(field) && (*mode == GenMode::Get || *mode == GenMode::GetMut) {
+            if has_prefix_attr(field)
+                && (*mode == GenMode::Get || *mode == GenMode::GetCopy || *mode == GenMode::GetMut)
```
Comment (djc) [inline src/generate.rs]: This is getting a little ugly, maybe add a `prefix()` method to `GenMode` that does the right thing?

Comment (bachue) [reply src/generate.rs]: @djc resolved

**Comment by Hoverbear** [comment]
Failing CI is because Travis windows is bad at caching, merging.

**Pull request merged**